src/integrationTest/groovy/wooga/gradle/releaseNotesGenerator/IntegrationSpec.groovy

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 the original author or authors.
```

Comment:
is the copyright notice 'author or authors' intended?

Reply:
A well I see. I also need to update the main License file.